guide/bioloid.md

```diff
@@ -0,0 +1,50 @@
+## Robotis's Bioloid
+
+%figure "A Bioloid dog-like model in Webots"
+
+![model.png](images/robots/bioloid/model.png)
+
+%end
+
+The "Bioloid Comprehensive Kit" allows to create articulated robots.
+This kit is developed by [Robotis](http://www.robotis.com) and is commercially available from [Tribotix](http://www.tribotix.com).
+
```

Comment:
Instead of mentioning one of the distributors, I would refer to the wikipedia page: https://en.wikipedia.org/wiki/Robotis_Bioloid